jest.config.js

```diff
@@ -25,20 +25,18 @@ module.exports = {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  collectCoverageFrom: [
-    'src/js/**/*.{js,jsx,ts,tsx}',
-    '!src/js/index.tsx',
-    '!src/js/sentry.ts',
-    '!src/js/i18n.ts',
-  ],
+  collectCoverageFrom: ['src/js/**/*.{js,jsx,ts,tsx}'],
 
   // The directory where Jest should output its coverage files
   coverageDirectory: 'coverage',
 
   // An array of regexp pattern strings used to skip coverage collection
-  // coveragePathIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
+  coveragePathIgnorePatterns: [
+    'src/js/index.tsx',
+    'src/js/sentry.ts',
+    'src/js/i18n.ts',
```

Comment:
I'm a little leery of ignoring `src/js/index.tsx` and `src/js/i18n.ts` as I expect them to have meaningful code in them; is the only way to exercise them to run the whole project or something?

Reply:
Hm, I could see the argument there for `index.tsx` at least. It's not a change in this PR, but I'll consider how difficult that would be.